### ISSUE TITLE
Launch KCCNC EU Transparency Report

### ISSUE THREAD

**Issue by thetwopct** [cncf/cncf.io#743]
### IMAGES
- [x] Upload images to production and pull in image ids
- [x] Check correct images being used
- [x] Lazy load images
- [x] Alt text is descriptive
- [x] Image dimensions w x h

### LINKS
- [x] Add all links where needed
- [x] Check anchor links
- [x] Broken links
- [x] Set title text on links

### TEXT
- [x] Check max widths match Figma
- [x] Copy and paste text in spell checker, look for mistakes
- [x] Check brand spelling, capitals, + not and/&. KubeCon + CloudNativeCon.

### CONFORMANCE
- [x] Run through accessibility checker
- [x] Run through W3C checker
- [ ] Browser Testing - last 2 versions
- [x] Check Lighthouse score for any other issues

### GO LIVE
- [x] Merge
- [x] Add Social Share image / report cover
- [x] Post preview link in Slack/Git issue
- [x] Post final permalink link in Slack/Git issue

**Comment by thetwopct** [comment]
This is live, but requires some final checking before closing 
https://www.cncf.io/reports/kubecon-cloudnativecon-europe-2023/

**Comment by cjyabraham** [comment]
WCAG and WC3 issues are now fixed and deployed.
